src/paulscode/android/mupen64plusae/input/map/TouchMap.java

```diff
@@ -265,6 +271,28 @@ public int getButtonPress( int xLocation, int yLocation )
     }
     
     /**
+     * Gets the frame for the N64 button with a given asset name
+     * 
+     * @param assetName The asset name for the button
+     * 
+     * @return The frame for the N64 button with the given asset name
+     * 
+     */
+    public Rect getButtonFrame( String assetName )
```

Comment:
This would work for any Image instance, buttons or analog.

I'll take a peek at the demapping... there may already be a static readonly map somewhere for this need.  If not, maybe it would be good to do it with a map for parallelism with the other code.  I'll have to refresh my memory... it's been a long time since I looked at this part of the code.